models/public/yolact-resnet50-fpn-pytorch/yolact-resnet50-fpn-pytorch.md

```diff
@@ -0,0 +1,99 @@
+# yolact-resnet50-fpn-pytorch
+
+## Use Case and High-Level Description
+
+YOLACT ResNet 50 is a simple, fully convolutional model for real-time instance segmentation described in "YOLACT: Real-time Instance Segmentation" [paper](https://arxiv.org/abs/1904.02689). Model pretrained in Pytorch\* on COCO dataset.
+For details, see the [repository](https://github.com/dbolya/yolact).
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Instance segmentation                     |
+| GFlops                          | 118.575                                   |
+| MParams                         | 36.829                                    |
+| Source framework                | Pytorch\*                                 |
+
+## Accuracy
+
+| Metric | Value |
+| ------ | ----- |
+| AP@masks | 28.00%|
+| AP@boxes |30.69%|
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `input.1`, shape: [1x3x550x550], format: [BxCxHxW],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: `RGB`.
+   Mean values - [123.675,116.28,103.53], scale values - [58.395,57.12,57.375].
+   
+
+### Converted Model
+
+Image, name: `input.1`, shape: [1x3x550x550], format: [BxCxHxW],
+where:
+
+   - B - batch size
+   - C - number of channels
+   - H - image height
+   - W - image width
+
+Expected color order: `BGR`.
+
+
+## Output
+
+### Original Model
+
+1. Detection scores, name: `conf`. Contains predicted bounding-boxes classes in a range [0, 80]. The model was trained on the Microsoft\* COCO dataset version with 80 categories of objects, 0 class is for background. Output shape is [1, 19248, 81] in [B, N, C] format, where
```

Comment:
```suggestion
1. Detection scores, name: `conf`. Contains score distribution over all classes in the [0,1] range . The model was trained on the Microsoft\* COCO dataset version with 80 categories of objects, 0 class is for background. Output shape is [1, 19248, 81] in [B, N, C] format, where
```

Reply:
Or they may be in logit format, you  should know better

Reply:
not rememer exactly, I made support several months ago, but if I dight remember I aplied sigmoid only on masks, scores used as is

Reply:
yes, they are in [0, 1] range